src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -78,13 +80,28 @@ class OverviewRoute extends React.Component<OverviewRouteProps, State> {
       <>
         <Row>
           <Col sm={colNum}>
-            {showMarketInsights && (
-              <>
-                <MarketInsights artist={artist} />
-                <Spacer mb={1} />
-              </>
-            )}
-
+            <ContextConsumer>
+              {({ user }) => {
+                if (
+                  user.type === "Admin" &&
```

Comment:
If the user is logged out, then `user` will be undefined so this should be `if(user && user.type === "Admin")`.